ext/mysqli/tests/mysqli_class_mysqli_result_interface.phpt

```diff
@@ -130,14 +130,10 @@ require_once 'skipifconnectfailure.inc';
         $mode = mt_rand(-1000, 1000);
     } while (in_array($mode, $valid));
 
-    if ($TEST_EXPERIMENTAL) {
-        ob_start();
-        $res = new mysqli_result($link, $mode);
-        $content = ob_get_contents();
-        ob_end_clean();
-        if (!stristr($content, 'Invalid value for resultmode'))
-            printf("[009] Expecting warning because of invalid resultmode\n");
-    }
+    try {
+        new mysqli_result($link, $mode);
+        printf("[009] Expecting ValueError because of invalid resultmode\n");
+    } catch (ValueError $ex) {}
```

Comment:
Why not add the exception message to the EXPECT section instead?

Reply:
I was just following the usual mysqli test conventions (and what was already there). I can change it, if you want me to.

Reply:
Yes, please do. It will be better.

Reply:
Done.